packages/ounit/ounit.2.2.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Sylvain Le Gall <sylvaini+ocaml@le-gall.net>"
+authors: [ "Maas-Maarten Zeeman" "Sylvain Le Gall" ]
+homepage: "https://github.com/gildor478/ounit"
+dev-repo: "git+https://github.com/gildor478/ounit.git"
+bug-reports: "https://github.com/gildor478/ounit/issues"
+doc: "https://gildor478.github.io/ounit/"
+depends: [
+  "ocamlfind"
+  "ounit2" {= version}
+]
+build: [
+  [make "null"]
```

Comment:
What is the goal of that? This command does nothing from what i can see

Reply:
I was unsure if you need a build step or not and got even less sure because of some errors in dune-release. Fixing it.